CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 1.1.2
```

Comment:
Maybe this should be `1.2.0`, as this is a new "feature"?

Reply:
I've got the same doubt, so I went with `1.1.2`. But I'm happy you got the objection, I'll fix it